contracts/implementations/erc20Pay/RMRKEquippableImplErc20Pay.sol

```diff
@@ -103,4 +103,19 @@ contract RMRKEquippableImplErc20Pay is
     function _charge(uint256 value) internal virtual override {
         _chargeFromToken(msg.sender, address(this), value);
     }
+
+    /**
+     * @notice Used to withdraw the minting proceedings to a specified address.
+     * @dev This function can only be called by the owner.
+     * @param erc20 Address of the ERC20 token to withdraw
+     * @param to Address to receive the given amount of minting proceedings
+     * @param amount The amount to withdraw
+     */
+    function withdrawRaisedERC20(
+        address erc20,
+        address to,
+        uint256 amount
+    ) external onlyOwner {
```

Comment:
I was considering whether we should add a reentrancy guard here as it is an external call 🤔

But I think we should be ok.. Even if the `erc20` contract is malicious and attempts to drain another token using `delegatecall`, the `msg.sender` would be this smart contract's address, so it would revert on `onlyOwner` check 🤔

Reply:
Yeah this feels pretty safe